harness/testIntl.js

```diff
@@ -2278,6 +2279,7 @@ var numberingSystemDigits = {
   takr: "𑛀𑛁𑛂𑛃𑛄𑛅𑛆𑛇𑛈𑛉",
   talu: "᧐᧑᧒᧓᧔᧕᧖᧗᧘᧙",
   tamldec: "௦௧௨௩௪௫௬௭௮௯",
+  tnsa: "\u16AC0\u16AC1\u16AC2\u16AC3\u16AC4\u16AC5\u16AC6\u16AC7\u16AC8\u16AC9",
```

Comment:
It's 5 digits so it requires braces:
```suggestion
  tnsa: "\u{16AC0}\u{16AC1}\u{16AC2}\u{16AC3}\u{16AC4}\u{16AC5}\u{16AC6}\u{16AC7}\u{16AC8}\u{16AC9}",
```
(`\u16AC0` is `\u16AC` followed by `0`)